libraries/AP_Camera/AP_Camera_config.h

```diff
@@ -13,6 +14,10 @@
 #define AP_CAMERA_BACKEND_DEFAULT_ENABLED AP_CAMERA_ENABLED
 #endif
 
+#ifndef AP_CAMERA_SEND_FOV_STATUS_ENABLED
```

Comment:
Please move this above the definition of `AP_CAMNERA_BACKEND_DEFAULT_ENABLED`

Reply:
I have done. but why was this required ?

Reply:
> I have done. but why was this required ?

The default value is used for all of the backends, so putting something between it and the backend stuff isn't clean.